README.md

```diff
@@ -1,105 +1,55 @@
-<!-- AUTO-GENERATED-CONTENT:START (STARTER) -->
-<p align="center">
-  <a href="https://www.gatsbyjs.com">
-    <img alt="Gatsby" src="https://www.gatsbyjs.com/Gatsby-Monogram.svg" width="60" />
-  </a>
-</p>
-<h1 align="center">
-  Gatsby's blog starter
-</h1>
+# Quarkus Extensions Explorer
 
-Kick off your project with this blog boilerplate. This starter ships with the main Gatsby configuration files you might need to get up and running blazing fast with the blazing fast app generator for React.
+This site is built using Gatsby, and hosted using GitHub pages.
 
-_Have another more specific idea? You may want to check out our vibrant collection of [official and community-created starters](https://www.gatsbyjs.com/docs/gatsby-starters/)._
+# Local development 
 
-## 🚀 Quick start
+Local development is a Quarkus-like experience, with continuous testing and live reload. 
+You will need node and npm set. 
 
-1.  **Create a Gatsby site.**
+Using nvm is an easy way to manage node versions:
+```
+brew install nvm
+export NVM_DIR="$HOME/.nvm"
+  [ -s "/opt/homebrew/opt/nvm/nvm.sh" ] && \. "/opt/homebrew/opt/nvm/nvm.sh"  # This loads nvm
+  [ -s "/opt/homebrew/opt/nvm/etc/bash_completion.d/nvm" ] && \. "/opt/homebrew/opt/nvm/etc/bash_completion.d/nvm"  # This loads nvm bash_completion
+```
 
-    Use the Gatsby CLI ([install instructions](https://www.gatsbyjs.com/docs/tutorial/part-0/#gatsby-cli)) to create a new site, specifying the blog starter.
+Check the last two lines against the suggested commands in the output of the `nvm` install. 
 
-    ```shell
-    # create a new Gatsby site using the blog starter
-    gatsby new my-blog-starter https://github.com/gatsbyjs/gatsby-starter-blog
-    ```
+```
+nvm install 14
+nvm use 14
+```
 
-1.  **Start developing.**
+In one terminal, run tests
+```
+npm run test:watch
```

Comment:
this gives me:

```
npm run test:watch

> quarkus-extensions@ test:watch /Users/manderse/code/quarkusio/extensions.quarkus.io
> ACTIVE_ENV=test SUPPRESS_ENV_OUTPUT=true jest --watch --testPathIgnorePatterns test-integration/

sh: jest: command not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! quarkus-extensions@ test:watch: `ACTIVE_ENV=test SUPPRESS_ENV_OUTPUT=true jest --watch --testPathIgnorePatterns test-integration/`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the quarkus-extensions@ test:watch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/manderse/.npm/_logs/2022-11-16T10_12_04_227Z-debug.log
```

Reply:
Forgot the `npm install`. Pushed update.